apps/glusterfs/block_settings.go

```diff
@@ -13,5 +13,6 @@ var (
 	// Default block settings
 	CreateBlockHostingVolumes = false
 	// Default 1 TB
-	BlockHostingVolumeSize = 1024
+	BlockHostingVolumeSize    = 1024
+	BlockHostingVolumeOptions = "group gluster-block"
```

Comment:
If ever one chooses to have a vanilla volume for block hosting (for any reason, experiment or otherwise) they will have to read the list of options in gluster-block and set them with default values.

I assumed that users would add additional options on command line using the gluster volume options feature.